uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveBalanceChange.php

```diff
@@ -822,6 +822,7 @@ private static function getBalanceChangesToExpire() {
    * over a given period of time.
    *
    * @param int $contactID
+   * @param int $absenceTypeId
```

Comment:
can you please rename this to `$abseceTypeID`, just to keep it consistent with `$contactID`?